mmv1/products/workbench/Instance.yaml

```diff
@@ -0,0 +1,471 @@
+        immutable: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: email
+              default_from_api: true
+              description: Optional. Email address of the service account.
+            - !ruby/object:Api::Type::Array
+              name: scopes
+              item_type: Api::Type::String
+              default_from_api: true
+              description: |
+                Output only. The list of scopes to be made available for this
+                service account. Set by the CLH to https://www.googleapis.com/auth/cloud-platform
+              output: true
+          description: |
+            Optional. The service account that serves as an identity for the
+            VM instance. Currently supports only one service account.
+      - !ruby/object:Api::Type::NestedObject
+        name: vmImage
+        default_from_api: true
+        immutable: true
+        custom_flatten: templates/terraform/custom_flatten/workbench_instance_vm_image_flatten.go.erb
+        description: |
+          Definition of a custom Compute Engine virtual machine image for starting
+          a workbench instance with the environment installed directly on the VM.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: project
+            description: |
+                The name of the Google Cloud project that this VM image belongs to.
+                Format: {project_id}
+          - !ruby/object:Api::Type::String
+            name: name
+            description: Optional. Use VM image name to find the image.
+            exactly_one_of:
+              - vm_image.0.name
+              - vm_image.0.family
+          - !ruby/object:Api::Type::String
+            name: family
+            description: |
+              Optional. Use this VM image family to find the image; the newest
+              image in this family will be used.
+            exactly_one_of:
+              - vm_image.0.name
+              - vm_image.0.family
+      - !ruby/object:Api::Type::NestedObject
+        name: bootDisk
+        default_from_api: true
+        description: The definition of a boot disk.
+        immutable: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: diskSizeGb
+            default_from_api: true
+            immutable: true
+            description: |
+              Optional. The size of the boot disk in GB attached to this instance,
+              up to a maximum of 64000 GB (64 TB). If not specified, this defaults to the
+              recommended value of 150GB.
+          - !ruby/object:Api::Type::Enum
+            name: diskType
+            default_from_api: true
+            immutable: true
+            custom_flatten: templates/terraform/custom_flatten/workbench_instance_boot_disk_type_flatten.go.erb
+            values:
+              - DISK_TYPE_UNSPECIFIED
```

Comment:
Unspecified should probably be removed, it's almost never allowed for a client to set